app/src/main/java/com/hedvig/app/feature/birthday/BirthdayActivity.kt

```diff
@@ -0,0 +1,70 @@
+package com.hedvig.app.feature.birthday
+
+import android.animation.Animator
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.hedvig.app.R
+import com.hedvig.app.feature.loggedin.ui.LoggedInActivity
+import com.hedvig.app.util.extensions.view.hide
+import com.hedvig.app.util.extensions.view.show
+import kotlinx.android.synthetic.main.activity_birthday.*
+import timber.log.Timber
+
+
+class BirthdayActivity : AppCompatActivity() {
```

Comment:
We can use the following new constructor to not have to add `setContentView(...)` in `onCreate()` 🙂 

```suggestion
class BirthdayActivity : AppCompatActivity(R.layout.activity_birthday) {
```